tests failed
```
======================================================================
FAIL: test_escape_output_value (corehq.apps.app_manager.tests.test_translations.AppManagerTranslationsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_translations.py", line 31, in test_escape_output_value
    self.assertEqual(expected_output, etree.tostring(escape_output_value(input)))
AssertionError: '<value>Here is a ref <output value="/data/no_media"/> with some "trailing" text &amp; that\'s some bad &lt; xml.</value>' != '<value>Here is a ref &lt;output value="/data/no_media" /&gt; with some "trailing" text &amp; that\'s some bad &lt; xml.</value>'
----------------------------------------------------------------------
```

--- a/corehq/apps/app_manager/tests/test_translations.py
+++ b/corehq/apps/app_manager/tests/test_translations.py
@@ -23,8 +23,10 @@ class AppManagerTranslationsTest(TestCase, SuiteMixin):
             ('unfortunate <xml expression', '<value>unfortunate &lt;xml expression</value>'),
             (u'क्लिक', '<value>&#2325;&#2381;&#2354;&#2367;&#2325;</value>'),
             ('&#39', '<value>&amp;#39</value>'),
-            ('Here is a ref <output value="/data/no_media" /> with some "trailing" text & that\'s some bad < xml.',
-             '<value>Here is a ref <output value="/data/no_media"/> with some "trailing" text &amp; that\'s some bad &lt; xml.</value>')
+            ('question1 is <output value="/data/question1" vellum:value="#form/question1"/> !',
+             '<value>question1 is &lt;output value="/data/question1" vellum:value="#form/question1"/&gt; !</value>'),
+            ('Here is a ref <output value="/data/no_media"/> with some "trailing" text & that\'s some bad < xml.',
+             '<value>Here is a ref &lt;output value="/data/no_media"/&gt; with some "trailing" text &amp; that\'s some bad &lt; xml.</value>')
 
         ]
         for input, expected_output in test_cases:

